frontend/src/scenes/plugins/pluginsLogic.ts

```diff
@@ -158,6 +159,17 @@ export const pluginsLogic = kea<
                 },
             },
         ],
+        personalApiKey: [
+            [] as PersonalAPIKeyType[],
+            {
+                createKey: async (label: string) => {
+                    const newKey: PersonalAPIKeyType = await api.create('api/personal_api_keys/', {
+                        label,
+                    })
+                    return newKey
+                },
+            },
```

Comment:
The default value is an `[]` array of type `PersonalAPIKeyType[]`, yet the loader returns a single `PersonalAPIKeyType`